### DISCUSSION TITLE
FR: Change standard Status field type to radio buttons offering immediate view of options (including colors)

### DISCUSSION THREAD

**Discussion by pyrobob83** [ExpressionEngine/ExpressionEngine#2856] [Feature Requests]
Started from a thread from Slack - the desire is to update the standard Status dropdown when editing entries be inline radio buttons that immediately show all options and their corresponding color schemes instead of the current dropdown which adds an unnecessary second click and obscures the options/colors until that first click.

![image](https://user-images.githubusercontent.com/8193519/174872354-626981fe-c577-4e4c-8cd4-dba96484d55f.png)

**Comment by intoeetive** [comment]
@ajohnson FYI

**Comment by murtaugh** [comment]
+1

I like this idea if only because the color coding system is completely missing in the editing workflow.

**Comment by intoeetive** [comment]
@TomJaeger if I recall correctly, we _temporary_ removed color coding because the dropdown component could not render it properly.
Maybe we should consider fixing that - or switching to radio buttons as suggested here

**Comment by pyrobob83** [comment]
FWIW I'd still prefer radios here vs trying to get colors in the dropdown because of the immediate visibility of all options.

**Comment by chunkitdesign** [comment]
+1 Please consider having it implemented.

**Comment by intoeetive** [comment]
https://github.com/ExpressionEngine/ExpressionEngine/pull/3917 is adding the color labels
